internal/services/applications/parse/known_clients.go

```diff
@@ -69,3 +69,13 @@ func (id KnownClientsId) Segments() []resourceids.Segment {
 func (id KnownClientsId) String() string {
 	return fmt.Sprintf("Known Clients (Application ID: %q)", id.ApplicationId)
 }
+
+func (id KnownClientsId) FromParseResult(input resourceids.ParseResult) error {
```

Comment:
This'll need to be a pointer:

```suggestion
func (id *KnownClientsId) FromParseResult(input resourceids.ParseResult) error {
```